www/index.html.tmpl

```diff
@@ -8,6 +8,11 @@
             #variables > div {
                 margin-bottom: 10px;
             }
+
+            input.error {
+                background-color: #fcc;
```

Comment:
pls use a different bg color in protest of FCC's net neutrality repeal

Reply:
lol I actually took this from RT's reply background, since I thought it made it pretty clear while making the text still readable enough.

Reply:
That being said, it's only red while there's no text in it, so that doesn't matter much, I'll make it more red.